### ISSUE TITLE
UI DESIGN : Landing page 

### ISSUE THREAD

**Issue by akashamin01** [oss2019/conversational-forms#2]
## purpose
Design a good landing page, for the people to access the service, the project is more of a Web based form implementation hence we will be preparing a web based service and hence for this we need to design a landing page, which talks about our product and provides a basic documentation on how one can implement the form with their use to get responce from the user.

ping me at : aakashjamin1239@gmail.com or on the [gitter channel](https://gitter.im/oss2019/conversational-forms) if you are willing to work on the issue

**Comment by Hrithik28** [comment]
I am interested in working on this issue.

**Comment by akashamin01** [comment]
sure ping me on the gitter channel i will send you the UI transormation and design for now 
@Hrithik28

**Comment by sourabhpruthi10** [comment]
@akashamin01 i will work on this . please give me some links to get started

**Comment by geekayush** [comment]
@akashamin01 Is this project active?

**Comment by akashamin01** [comment]
yes this project is active as for now we are done with the UI we haven't commited anything as the UI person we not much familier with github, now we are working on backend(DB), and NLP majorly django python, if guys are intersted ping me on gitter i will brief you about the ongoing things and responsiblities.
